ARM-wvd-templates/AddVirtualMachinesToHostPool/AddVirtualMachinesTemplate.json

```diff
@@ -409,6 +409,27 @@
         "description": "System data is used for internal purposes, such as support preview features."
       },
       "defaultValue": {}
+    },
+    "securityType": {

```

Comment:
If the only valid value for this is an empty string or TrustedLaunch then you should specify "allowedValues" with those values so the customer will get an error if they enter something else.
This applied to these same parameters in the other template files in this PR too.